server/game/cards/01_SOR/ImperialInterceptor.ts

```diff
@@ -0,0 +1,31 @@
+import AbilityHelper from '../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../core/card/NonLeaderUnitCard';
+import { Location, RelativePlayer, WildcardCardType } from '../../core/Constants';
+
+export default class ImperialInterceptor extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '9002021213',
+            internalName: 'imperial-interceptor'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'You may deal 3 damage to a space unit.',
```

Comment:
nit, consistency: we typically aren't using a period at the end of titles / messages like this

Reply:
no problem